### PR TITLE
Fixed broken urls

### PR DESCRIPTION
**Pull request by Jlaird**
Currently they look like this (below) that don't work, I updated them to look to the local file.
https://github.com/endeavorb2b/websites/blob/master/blob/master/.travis.yml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,14 +59,14 @@ Additionally, we recommend the following tools/utilities:
   - All project linting rules should be followed (see ESLint at project root).
   - While all ES6 code will be transpiled for the browser accessing it at run-time, beware of any potential deprecations that are not included in the transpilation.
     - Utilize [CanIUse](https://caniuse.com) to check targeted browser support for native JS and CSS methods.
-    - If an additional polyfill is required, make sure the [loader](https://github.com/base-cms/base-cms/blob/master/packages/marko-web/src/components/document/index.marko#L11) is updated to reflect this.
```
Comment (solocommand) [inline README.md]: This one is already fully pathed and now 404s

File: README.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ Packages should use dasherized folder names (`my-package` rather than `myPackage
 Sites should match the lowercase domain name of the site exactly, excluding the GTLD (.com/etc) or any subdomain (`www.my-NEW-site.com.jp` should be added as `my-new-site`.)
 
 ### Creating a new site
-To add a new site to this repository, you first need to know the Base Platform infrastructure stack. You can find this in the relevant [groups](https://github.com/cygnusb2b/base-platform/blob/master/app/config/platform/groups/as3.yml) definition under the `infrastructure_group` key.
```
Comment (solocommand) [inline README.md]: Same here

**Comment by Jlaird** [comment]
@solocommand I updated the links you pointed out.

**Comment by zarathustra323** [comment]
@solocommand Are you good with these?

**Pull request merged**